scripts/browserifyTestDependency.js

```diff
@@ -0,0 +1,66 @@
+/*!
+Copyright 2016 OCAD University
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+The research leading to these results has received funding from the European Union's
+Seventh Framework Programme (FP7/2007-2013) under grant agreement no. 289016.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+// The creation of this script is to work around the issue of having to run a npm postinstall step
+// only within the context of a GPII development environment.
+// This script uses fluid-resolve npm package (https://www.npmjs.com/package/fluid-resolve) to detect
+// the existence of %universal/node_modules/browserify directory that is only pulled in as a devDependency.
+// If the directory is present, this script browserifies some node js scripts that are used by some
+// in-browser tests. Otherwise, skip the browserifying and exit quitely.
+
+"use strict";
+
+var fluid = require("infusion"),
+    resolve = require("resolve"),
+    rimraf = require("rimraf"),
+    mkdirp = require("mkdirp"),
+    exec = require('child_process').exec;
+
+var universalPath = fluid.module.resolvePath("%universal");
+
+// Detect whether the devDependency module "browserify" exists
+resolve("browserify", {basedir: universalPath}, function (err, res) {
+    if (err) {
+        fluid.log("GPII is not running in a development mode, skipped browserifying test dependent node js scripts.")
```

Comment:
fluid.log is a bad choice for infrastructure scripts since it defaults to throwing the output away. A console.log here is more appropriate